package.json

```diff
@@ -1,139 +1,30 @@
-    "@babel/eslint-parser": "^7.21.3",
-    "@ember/optional-features": "^2.0.0",
-    "@ember/string": "^3.0.1",
-    "@ember/test-helpers": "^3.2.0",
-    "@embroider/test-setup": "^3.0.2",
-    "@glimmer/component": "^1.1.2",
-    "@glimmer/tracking": "^1.1.2",
-    "@release-it-plugins/lerna-changelog": "^6.0.0",
-    "@tsconfig/ember": "^3.0.1",
-    "@types/ember": "^4.0.3",
-    "@types/ember-data": "^4.4.8",
-    "@types/ember-data__adapter": "^4.0.1",
-    "@types/ember-data__model": "^4.0.0",
-    "@types/ember-data__serializer": "^4.0.1",
-    "@types/ember-data__store": "^4.0.2",
-    "@types/ember-resolver": "^9.0.0",
-    "@types/ember__application": "^4.0.5",
-    "@types/ember__array": "^4.0.3",
-    "@types/ember__component": "^4.0.12",
-    "@types/ember__controller": "^4.0.4",
-    "@types/ember__debug": "^4.0.3",
-    "@types/ember__destroyable": "^4.0.1",
-    "@types/ember__engine": "^4.0.4",
-    "@types/ember__error": "^4.0.2",
-    "@types/ember__object": "^4.0.5",
-    "@types/ember__polyfills": "^4.0.1",
-    "@types/ember__routing": "^4.0.12",
-    "@types/ember__runloop": "^4.0.2",
-    "@types/ember__service": "^4.0.2",
-    "@types/ember__string": "^3.0.10",
-    "@types/ember__template": "^4.0.1",
-    "@types/ember__test": "^4.0.1",
-    "@types/ember__test-helpers": "^2.9.1",
-    "@types/ember__utils": "^4.0.2",
-    "@types/qunit": "^2.19.4",
-    "@types/rsvp": "^4.0.4",
-    "concurrently": "^8.0.1",
-    "ember-cli": "~4.11.0",
-    "ember-cli-dependency-checker": "^3.3.1",
-    "ember-cli-htmlbars": "^6.2.0",
-    "ember-cli-inject-live-reload": "^2.1.0",
-    "ember-cli-sri": "^2.1.1",
-    "ember-cli-terser": "^4.0.2",
-    "ember-data": "^5.3.0",
-    "ember-load-initializers": "^2.1.2",
-    "ember-page-title": "^8.0.0",
-    "ember-qunit": "^7.0.0",
-    "ember-resolver": "^11.0.1",
-    "ember-source": "~5.3.0",
-    "ember-source-channel-url": "^3.0.0",
-    "ember-template-lint": "^5.7.2",
-    "ember-try": "^3.0.0",
-    "eslint": "^8.37.0",
-    "eslint-config-prettier": "^9.0.0",
-    "eslint-plugin-ember": "^11.4.9",
-    "eslint-plugin-n": "^16.2.0",
-    "eslint-plugin-prettier": "^5.0.1",
-    "eslint-plugin-qunit": "^8.0.1",
-    "loader.js": "^4.7.0",
-    "prettier": "^3.0.3",
-    "qunit": "^2.20.0",
-    "qunit-dom": "^3.0.0",
-    "release-it": "^16.2.1",
-    "sinon": "^16.1.0",
-    "typescript": "^5.0.3",
-    "webpack": "^5.89.0"
-  },
-  "peerDependencies": {
-    "ember-source": ">=3.28.0",
-    "qunit": "^2.0.0",
-    "sinon": "^15.0.3 || ^16.0.0"
-  },
-  "engines": {
-    "node": "14.* || 16.* || >= 18"
-  },
-  "volta": {
-    "node": "18.12.0",
-    "yarn": "1.22.19"
-  },
-  "publishConfig": {
-    "registry": "https://registry.npmjs.org"
-  },
-  "ember": {
-    "edition": "octane"
-  },
-  "ember-addon": {
-    "configPath": "tests/dummy/config",
-    "after": [
-      "ember-sinon"
-    ]
+    "concurrently": "^8.2.1",
+    "@release-it-plugins/lerna-changelog": "^6.0.0"
```

Comment:
What's the reasoning to split up this plugin dependency from the release-it dependency?

Reply:
ah good catch! We indeed need to have `release-it` in root level package.json in `devDependencies`.

Reply:
@SergeAstapov, @elwayman02 my mistake, it should be fixed now